src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Assessment.kt

```diff
@@ -32,7 +33,7 @@ data class Assessment(
   @NotNull
   val id: Long = -1,
 
-  @ManyToOne(optional = false)
+  @ManyToOne(optional = false, fetch = FetchType.LAZY)
```

Comment:
I think for caseloads we generally bulk load assessments and their associated offender - does this turn that into an n+1?

Reply:
Will change back to Eager